bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/i18n/avmfritz.properties

```diff
@@ -137,6 +137,10 @@ channel-type.avmfritz.comfort_temp.label = Comfort Temperature
 channel-type.avmfritz.comfort_temp.description = Thermostat Comfort temperature.
 channel-type.avmfritz.contact_state.label = Contact State
 channel-type.avmfritz.contact_state.description = Contact state information (OPEN/CLOSED).
+channel-type.avmfritz.obstruction_alarm.label = Obstruction Alarm
+channel-type.avmfritz.obstruction_alarm.description = Obstruction alarm of the blinds. The blinds were stopped and moved a bit in the opposite direction.
+channel-type.avmfritz.temperature_alarm.label = Temperature Alarm
+description = Temperature alarm of the blinds. Indicates overheating of the motor.
```

Comment:
Something is missing at the beginning of that line.

Reply:
Repaired. Thanks for spotting this.